test/collection-browser.test.ts

```diff
@@ -227,6 +228,79 @@ describe('Collection Browser', () => {
     ).to.contains('Results');
   });
 
+  it('fails gracefully if no search service provided', async () => {
+    const el = await fixture<CollectionBrowser>(
+      html`<collection-browser></collection-browser>`
+    );
+
+    el.baseQuery = 'collection:foo';
+    await el.updateComplete;
+
+    // This shouldn't throw an error
+    expect(el.fetchPage(2)).to.exist;
```

Comment:
Q: what does UI do?

Reply:
I believe it should just leave the empty placeholder up because the data source never gets updated if no search occurs. But I'll update the test to verify that!